crates/driver/solvers.example.toml

```
@@ -0,0 +1,12 @@
+[[solvers]]
+name = "mysolver" # Arbitrary name given to this solver, must be unique
+url = "http://localhost:1234/solve" # Endpoint of the solver
```

From the code, I think this is:
```suggestion
endpoint = "http://localhost:1234/solve" # Endpoint of the solver
```